### PR TITLE
add cdim property to audio classes

### LINKED ISSUE #602: Add `cdim` porperty to audio classes

### PR DESCRIPTION
**Pull request by hoyer-a**
Closes #602 

Added cdim property to audio classes

### CODE REVIEW

File: pyfar/classes/audio.py [hunk 1]
```diff
@@ -58,6 +58,17 @@ def cshape(self):
         """
         return self._data.shape[:-1]
 
+    @property
+    def cdim(self):
+        """
+        Return channel dimension.
```
Comment (f-brinkmann) [inline pyfar/classes/audio.py:64]:
```suggestion
        Return the channel dimension.
```

File: pyfar/classes/audio.py [hunk 2]
```diff
@@ -58,6 +58,17 @@ def cshape(self):
...
+
+        The channel dimension (cdim) is the length of the cshape. Gives number
+        of channel dimensions (e.g. ``self.cshape = (2, 3)``;
+        ``self.cdim = 2``).
```
Comment (f-brinkmann) [inline pyfar/classes/audio.py:68]:
```suggestion
        The channel dimension (`cdim`) is the length of the channel shape (`cshape`) (e.g. ``cshape = (2, 3)``;
        ``cdim = 2``).
```

**Review by f-brinkmann — APPROVED** [review]
Thanks. I made small suggestions to improve the wording - approved otherwise.

**Review by ahms5 — COMMENTED** [review]
Thank you for implementing. Just small docstring suggestions

File: pyfar/classes/audio.py (hunk 1 above)
Comment (hoyer-a) [reply pyfar/classes/audio.py:64]: In ``Coordinates`` it's the same for ``cdim``, i would leave it like that

**Review by ahms5 — DISMISSED** [review]

**Comment by ahms5** [comment]
is it a hotfix (merge into main) or a feature (merge into dev). Maybe we need to discuss on Friday.

**Comment by f-brinkmann** [comment]
Since we have it in other classes we could say were fixing a missing attribute. The next feature release will probably still take a while and it might be nice to have in the meantime. Bit I'm fine with both

**Pull request merged**